earth_enterprise/src/common/notify.cpp

```diff
@@ -132,12 +132,12 @@ std::string GetNotifyPrefixValue(khNotifyLevel severity, const std::string& noti
     if ((timeval != (time_t)-1) &&
         (localtime_r(&timeval, &ts) != 0) &&
         (strftime(buf, sizeof(buf), "%F %T", &ts) > 0)) {
-      sprintf(prefix_buf, "[%s] %s: ", buf, notifyLevels[severity]);
+      snprintf(prefix_buf, 1024, "[%s] %s: ", buf, notifyLevels[severity]);
     } else {
-      sprintf(prefix_buf, "[time] %s: ", notifyLevels[severity]);
+      snprintf(prefix_buf, 1024, "[time] %s: ", notifyLevels[severity]);
     }
   } else {
-    sprintf(prefix_buf, "%s %s:\t",
+    snprintf(prefix_buf, 1024, "%s %s:\t",
```

Comment:
Prefer const

Reply:
I used numeric literals because the same things are used to define all the buffers, I guess we could change everything to BUFLEN or something.